### PR TITLE
chore: update ui-kit dependencies

### PR DESCRIPTION
**Pull request by kark**

### CODE REVIEW

**Comment by kark** [comment]
Hey @chloe0592 👋,
could you please guide me what is the expected `size` prop value for the `<SecondaryIconButton>` used as the close button in all the dialogs? (You can observe the differences in VRTs). 
Should we now use `big` or `medium` that was used before? 
Or should I ask Filip?

File: packages/application-components/src/components/modal-pages/internals/modal-page-top-bar.tsx [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
+        height: 35px;
+        width: 48px;
+        top: 0;
+        right: 0;
+      }
+    `}
+  >
+    {cloneElement(props.children, { size: props.size })}
+  </span>
+);
```
Comment (kark) [inline packages/application-components/src/components/modal-pages/internals/modal-page-top-bar.tsx:41]: This is for the sake of the `size` prop drilling to the icon component

**Review by ddouglasz — APPROVED** [review]
looks good

**Comment by chloe0592** [comment]
> Hey @chloe0592 👋, ~could you please guide me what is the expected `size` prop value for the `<SecondaryIconButton>` used as the close button in all the dialogs? (You can observe the differences in the VRTs). Should we now use `big` or `medium` that was used before? Or should I ask Filip?~
> 
> Sorry, please disregard 👆 I think got it

@kark thank you for this change. ❤️  I should've done it but It slipped under my radar.

**Review by chloe0592 — APPROVED** [review]
Thank you! 🙇‍♀️

**Pull request merged**